### PR TITLE
feat: add NCBO BioPortal link

### PR DESCRIPTION
**Pull request by jsstevenson**
address #80 ? I didn't read the rest of the conversation, not sure if there's more work to be done there outside of this.

* Add button to link to corresponding NCBO BioPortal page
* Update other browser links to proper external links
* Special cases are all hard coded rather than figuring out a regex. It's implemented as a custom liquid plugin, should be fairly easy to update if necessary.

### CODE REVIEW

File: _plugins/bioportal_identifier.rb
Comment (matentzn) [inline _plugins/bioportal_identifier.rb:1]: Boom.

**Review by matentzn — APPROVED** [review]
Assuming it _works_ this looks greeeat!!! THANKS!

**Comment by matentzn** [comment]
@jsstevenson somehow the BioPortal links are lowercased on the site?

https://monarchinitiative.org/disease/MONDO:0019609

**Comment by jsstevenson** [comment]
@matentzn  ...bizarre. This was working locally. Give me a minute

**Comment by jsstevenson** [comment]
Oh, right, the way we're deploying the Jekyll site won't support custom liquid plugins. I'll need to do a quick fix

**Comment by nataled** [comment]
PRO already had a working BioPortal link. There are now two (the second, new link, is non-working).

**Comment by jsstevenson** [comment]
Ok, sorry about that. Fix for both is in https://github.com/OBOFoundry/OBOFoundry.github.io/pull/2655. This was working locally (although I missed the extra browser links for PRO et al) but GitHub's native jekyll deployment places some limits on the kinds of plugins you can add yourself.

**Pull request merged**